ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -597,6 +597,7 @@ objects:
         }
 
         function setup-google-cloud-sdk() {
+          pushd
```

Comment:
Not required to change (unless you want to) but you can do `pushd /tmp`.

Reply:
thx, changed